### DISCUSSION TITLE
in case of moving to new computer.

### DISCUSSION THREAD

**Discussion by geir75** [foldynl/QLog#426] [Q&A]
I'm trying to understand. Will I have to import newest autobackup .adx file if installing qlog on new computer? 
I understand qlog autobackps . adx format, located in ( Linux ):

 ~/.var/app/io.github.foldynl.QLog/data/hamradio/QLog

**Comment by muihlinn** [comment]
Last time I had to do that just moved the whole folder and it worked, except a few things, like password storage which I guess it's on a keychain whatever your system use.

**Reply by geir75** [reply]
Yea, I was acutally thinking about the same. Just replacing Qlog folder. Thanks for sharing your experience. 73.
